net-p2p/nicotine+/nicotine+-3.3.4.ebuild

```
@@ -0,0 +1,38 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+
+inherit distutils-r1 xdg
+
+DESCRIPTION="Graphical client for the Soulseek peer to peer network written in Python"
+HOMEPAGE="https://nicotine-plus.org/"
+SRC_URI="https://github.com/Nicotine-Plus/nicotine-plus/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/nicotine-plus-${PV}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+BDEPEND="sys-devel/gettext"
+RDEPEND="
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	|| ( >=x11-libs/gtk+-3.22.20:3[introspection] gui-libs/gtk:4[introspection] )
+"
+
+distutils_enable_tests pytest
+
+DOCS=( AUTHORS.md NEWS.md README.md TRANSLATORS.md )
+
+pkg_postinst() {
+	xdg_pkg_postinst
+
+	ewarn "Nicotine can work with both gtk3+ and gtk4."
+	ewarn "The never version is better but it has worse screen reader support"
```

```suggestion
	ewarn "The newer version is preferred but it has worse screen reader support"
```

Something like  that, it will use gtk4 if its available and you arent using a screenreader.